bin/sync-dev-deps.js

```diff
@@ -85,17 +87,34 @@ function updatePackageJson(pkgFile, masterDeps) {
     }
   }
   if (!modified) return false;
-  writeJsonFile(pkgFile, data);
+  writePackageJson(pkgFile, data);
   return true;
 }
 
 if (require.main === module) syncDevDeps();
 
-function readJsonFile(filePath) {
+function readPackageJson(filePath) {
   return JSON.parse(fs.readFileSync(filePath, 'utf-8'));
 }
 
-function writeJsonFile(filePath, data) {
+function writePackageJson(filePath, data) {
+  data.dependencies = sortObjectByKeys(data.dependencies);
+  data.devDependencies = sortObjectByKeys(data.devDependencies);
   fs.writeFileSync(filePath, JSON.stringify(data, null, 2) + '\n', 'utf-8');
   console.log('%s has been updated.', filePath);
 }
+
+/**
+ * Sort an object by keys
+ * @param data - An object to be sorted
+ */
+function sortObjectByKeys(data) {
+  if (data == null) return undefined;
```

Comment:
Do we need to check for `undefined` as well with strict equality operator `===`?

Reply:
`== null` will catch `null` and `undefined`.